### PR TITLE
Update toolkit to v4

### PR DESCRIPTION
**Pull request by Birden**

### CODE REVIEW

File: package-lock.json
Comment (buryndin) [inline package-lock.json:1]: Remove this file from repo

File: projects/seismic/src/lib/components/seismic.component.ts [hunk 1]
```diff
@@ -329,11 +330,11 @@ export class SeismicComponent extends AbstractComponent implements OnDestroy {
...
 
     // Default settings
-    const defaultOptions = {
+    const defaultOptions: DocumentExportSettings = {
       'paperformat': paper,
       'units': 'in',
-      'scaling': 'AsIs',
-      'asyncrendering': true
+      'scaling': ScalingOptions.AsIs,
+      // 'asyncrendering': true
```
Comment (buryndin) [inline projects/seismic/src/lib/components/seismic.component.ts]: remove

File: package.json [hunk 2]
```diff
@@ -58,7 +58,7 @@
     "shx": "^0.3.2",
     "ts-node": "~7.0.0",
     "tslint": "~6.1.0",
-    "typedoc": "^0.16.0",
+    "typedoc": "1.0.0-dev.4",
```
Comment (serrique) [inline package.json]: no

File: projects/welllog/src/lib/components/welllog-navigation/welllog-navigation.component.ts [hunk 3]
```diff
@@ -83,8 +83,8 @@ export class WellLogNavigationComponent extends AbstractComponent implements OnD
     this.widget.getTool().add(this._navigationTool);
 
 
-    this._navigationTool.addListener(NavigationEvents.DepthRangeChanged,
-      (sender, eventArgs) => {
+    this._navigationTool.on(NavigationEvents.DepthRangeChanged,
+      (event, sender, eventArgs) => {
         if (this.targetWellLog && this.targetWellLog.widget) {
           this.targetWellLog.widget.setVisibleDepthLimits(eventArgs['limits']);
```
Comment (buryndin) [inline projects/welllog/src/lib/components/welllog-navigation/welllog-navigation.component.ts]: It seems eventArgs['limits'] is deprecated

**Review by serrique — APPROVED** [review]

**Pull request merged**